integration/nevermined/SearchAsset.test.ts

```diff
@@ -84,4 +84,19 @@ describe('Search Asset', () => {
     assert.equal(ddos.length, 2)
     ddos.map((ddo) => assert.instanceOf(ddo, DDO))
   })
+
+  it('should be able get the assets by DID', async () => {
```

Comment:
this is searching by text, not DID

Reply:
```
nevermined.search.byDID(ddo.id).then((ddo) => assert.instanceOf(ddo, DDO))
```
what this is doing under the hood (via `retrieveDDO` function) is to compose the marketplace api url of the asset and get the DDO.
Why you mean is doing a search?

Reply:
I mean that the description of the test does not match the implementation

Reply:
sorry, didn't saw the call to `search.byDID`

Reply:
:+1: